acceptance-tests.md

```diff
@@ -100,12 +118,14 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
 
 1. **Add and track new hardware**
     * _As an administrator of the cluster, I should be able to add nodes to the cluster. I should also be able to view all the nodes and their status._
-    * <span style="text-decoration:underline;">Steps</span>
+    * Steps
         1. Netbox
+    * Acceptance tests: requires investigation
 2. **Track faulty hardware**
     * _As an administrator of the cluster, I should be able to view and track the list of faulty nodes that need to be replaced._
-    * <span style="text-decoration:underline;">Steps</span>
+    * Steps
         1. Nagios or refer to notes in Netbox
```

Comment:
Is there anything in ACM that allows you to track a list/re[rt of "faulty" nodes?  If so, that is relevant here.  Otherwise, this is another item for Justin.  At the very least, you want to make sure that we explain how to keep the OpenShift understanding of what working and infra servers are available in agreement with what hardware is actually available, and how to check on this.